docs/src/main/sphinx/installation/deployment.md

```diff
@@ -35,6 +35,10 @@ Trino requires a 64-bit version of Java 24, with a minimum required version of
 such as Java 8, Java 11, Java 17, Java 21 or Java 23 do not work.
 Newer versions such as Java 25 are not supported -- they may work, but are not tested.
 
+Trino requires enabling the [JDK Vector API extensions](https://openjdk.org/jeps/489) at runtime. Code using
```

Comment:
I don't think this is needed. The launcher enables it automatically, so there's nothing for users to do.

Reply:
Shall I rephrase this to make it clear the launcher already sets that flag, or just remove the section altogether? I've rephrased this section to make it more clear that the launcher sets the flag, but am happy to remove it altogether if you'd prefer.

Reply:
I would remove it altogether. It's a detail users shouldn't be concerned with -- we handle everything that's needed under the covers.

Reply:
I think it is good idea to mention it because we should showcase that we're actually using some cool new Java features. Also, developers launch Trino process without going through trino launcher and the information about adding the vector module as a requirement will avoid some confusion.

Reply:
Which developer section should I add a mention of this to?

Reply:
Opened https://github.com/trinodb/trino/pull/27342